internal/services/iothub/iothub_resource_test.go

```diff
@@ -1583,7 +1560,7 @@ resource "azurerm_iothub" "test" {
     purpose = "testing"
   }
 }
-`, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
+`, data.RandomInteger, "eastus", data.RandomInteger)
```

Comment:
I do see that this is a straight revert but we might want to either change it here or just remember to change these in the subsequent PR back to `data.Locations.Primary`